docs/source/design/v1/torch_compile.md

```diff
@@ -0,0 +1,131 @@
+# vLLM's `torch.compile` integration
+
+In vLLM's V1 architecture, `torch.compile` is enabled by default and is a critical part of the framework. This document gives a simple walk-through example to show how to understand the `torch.compile` usage.
+
+Throughout the example, we will run a common Llama model using v1, and turn on debug level logging to show all the details. The command to be used is `VLLM_USE_V1=1 VLLM_LOGGING_LEVEL=DEBUG vllm serve meta-llama/Llama-3.2-1B`.
+
+## Compilation Cache
+
+In the very verbose logs, we can see:
+
+```
+INFO 03-07 03:06:55 [backends.py:409] Using cache directory: ~/.cache/vllm/torch_compile_cache/1517964802/rank_0_0 for vLLM's torch.compile
+```
+
+vLLM will take all the available factors into consideration, and decide a directory to store all the compilation artifact. This means, you can directly copy the whole `~/.cache/vllm/torch_compile_cache` directory in your deployment scenario to save a great amount of compilation time, and hence accelerating the starting time of the vLLM instance.
```

Comment:
nit: May be add one sentence saying that we will explain how the compiled cache is hashed in the next section.

Reply:
added in [62c0ee3](https://github.com/vllm-project/vllm/pull/14437/commits/62c0ee3a8b8242ce241f8d6367669861efdb531b)